.github/workflows/image-health-check.yml

```diff
@@ -0,0 +1,90 @@
+name: Image Health Check
```

Comment:
I believe it would be better to include `update_image` in the file name as well as the `name:` attribute. This workflow do not actually check the health of existing images but tests if we can build images from scratch.

Reply:
@hanefi I see your point however I prefer approach the naming from why. The main reason we are creating the image is checking the health. When I see the name in Github, it makes sense to me. For other people as well